### DISCUSSION TITLE
smaller binary

### DISCUSSION THREAD

**Discussion by HookedBehemoth** [jpochyla/psst#329] [Ideas]
Did you consider getting the binary size down by stripping, enabling lto or changing the opt-level?
Running strip on the binary from https://nightly.link/jpochyla/psst/workflows/build/master already removes ~10.4MiB (28.2MiB -> 17.8MiB).
With my own config it's 12MiB.
```toml
[profile.release]
opt-level = 3
strip = true
lto = true
panic = "abort"
codegen-units = 1

**Comment by jpochyla** [comment]
Hi, thanks for the suggestion, I've totally forgotten about this!

**Comment by jacksongoode** [comment]
@Insprill Would be interesting to incorporate this whilst you're on your PR flurry 😆

**Comment by Insprill** [comment]
That didn't take long for someone to notice :P
jpochyla already did this in https://github.com/jpochyla/psst/commit/3c32645e23b2ce6ba4bdaa38c0707ae3336fe184 for the exception of changing the panic mode

**Reply by jacksongoode** [reply]
Ah ok great, good catch!
